src/foam/nanos/mrac/ClusterResponse.js

```diff
@@ -1,27 +1,27 @@
-/**
- * @license
- * Copyright 2019 The FOAM Authors. All Rights Reserved.
- * http://www.apache.org/licenses/LICENSE-2.0
- */
+// /**
```

Comment:
Delete this.
if you used git mv then it would have renamed this to ClusterOperation